docs/tutorials/java/background-check/project-setup.mdx

```diff
@@ -363,7 +363,7 @@ In the Temporal Java SDK programming model, a [Workflow Definition](https://docs
 #### Boilerplate Workflow Interface {#workflow-code}
 
 <!--SNIPSTART java-project-setup-chapter-boilerplate-workflow-interface-->
-[BackgroundCheckBoilerplateWorkflow.java](https://github.com/temporalio/temporal-learning/blob/main/docs/tutorials/java/background-check/code/backgroundcheck/src/main/java/backgroundcheckboilerplate/BackgroundCheckBoilerplateWorkflow.java)
+[BackgroundCheckBoilerplateWorkflow.java](code/backgroundcheck/src/main/java/backgroundcheckboilerplate/BackgroundCheckBoilerplateWorkflow.java)
```

Comment:
Did you mean to delete the first half of this url?

https://github.com/temporalio/temporal-learning/blob/main/docs/tutorials/java/background-check/code/backgroundcheck/src/main/java/backgroundcheckboilerplate/BackgroundCheckBoilerplateWorkflow.java

Reply:
Same for the ones below, why do we remove them?

Reply:
We remove them because the code was placed into the learn site and does not live on the repository. It's therefore treated as if we added an image folder as a subdirectory, a relative link.